bazel/fuzz_testing.bzl

```diff
@@ -21,12 +21,16 @@ DEFAULT_RUSTC_FLAGS = [
     "-Ccodegen-units=1",
     "-Zextra-const-ub-checks",
     "-Zstrict-init-checks",
+    "-Cstrip=none",
     # TODO(PSEC): Add configuration to enable only during profiling
     # "-Cinstrument-coverage",
 ]
 
 DEFAULT_SANITIZERS = [
     "-Zsanitizer=address",
+    # zig doesn't like how rustc pushes the sanitizers, so do it ourselves.
+    "-Zexternal-clangrt",
+    "-Clink-arg=bazel-out/k8-opt/bin/external/rust_linux_x86_64__x86_64-unknown-linux-gnu__nightly_tools/rust_toolchain/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc-nightly_rt.asan.a",
```

Comment:
is there a way to avoid using the full bazel build output path? that tends to break silently and cause hours of debugging

Reply:
Here's some discussion: https://github.com/dfinity/ic/pull/3508#discussion_r1984868869

We could patch to add the visibility? I wasn't able to figure out any other way to read directly into the toolchain, outside of a rule.